fabfile.py

```diff
@@ -396,7 +386,7 @@ def _kernel_deps():
         'xmlto',
         'libgtk2.0-dev',
 	'libunwind-setjmp0-dev',
-	'libunwind7-dev',
+#	'libunwind7-dev',
```

Comment:
so...why did we have this pre-xenial?  and if we needed it, how will pre-xenial continue to work?  and what takes its place on xenial?